docs-source/content/userguide/introduction/introduction.md

```diff
@@ -30,10 +32,16 @@ Detailed instructions are available [here]({{< relref "/userguide/managing-opera
   not need the `cluster-admin` role at runtime.
 * We do not currently support running WebLogic in non-Linux containers.
 
+For the current preview release 3.0.0-rc1, the above is modified as follows:
+
+* Kubernetes 1.13.5+ support is deprecated and removed.
+* Kubernetes 1.16.0+ support is added.
```

Comment:
Do we want to mention MII version requirements here?  1.8.1 for https://github.com/oracle/weblogic-deploy-tooling  and 1.8.5 for https:/github.com//oracle/weblogic-image-tool

Reply:
Might be better on the MII pages?  Since we would still support the others for non-MII domains right?

Reply:
Fine by me.  Up to you  if you want to put it there now, or have me put update later during v1.